components/CardDropdownComponent.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { styled } from "styled-components";
+import { useModalSetting } from "../hooks/useModalSetting";
+import CommonModalComponent from "./CommonModalComponent";
+
+type props = {
+  show: boolean;
+  url: string;
+};
+
+const CardDropdownComponent = ({ show, url }: props) => {
+  const [modalType, setModalType] = useState({
+    type: "",
+    title: "",
+    folderName: "",
+    linkName: "",
+  });
+  const { show: modalShow, modalOn, modalOff } = useModalSetting();
+  return (
```

Comment:
return문 전에도 개행을 넣어주는게 좋습니다~
```suggestion
  const { show: modalShow, modalOn, modalOff } = useModalSetting();
  
  return (
```